dist/s_docs

```diff
@@ -90,20 +90,6 @@ structurechk()
 	}
 }
 
-asciichk()
-{
-    # Make sure there are only ASCII characters in the input files.
-    # Odd characters can cause our Python filter to fail.
-    (cd ../src/docs && LC_ALL=C grep -n '[^[:print:]	]' *.dox | cat -v) > $t
```

Comment:
Do we no longer need this? Is it covered by what's in `s_style`?

Reply:
Yes, this work overlaps with what's covered in `s_style` now, as this outputs an error for non-ascii character in `.dox` files, and s_style will automatically remove the non-ascii characters in `.dox` files now.

Reply:
Thanks for the details @fionaochee !